src/main/resources/db/migration/changesets/1-create-tables.sql

```diff
@@ -1,9 +1,10 @@
 create table if not exists users (
   user_id serial primary key,
+  telegram_id bigint unique not null,
   username text unique not null,
-  firstname text,
+  firstname text not null,
   lastname text,
-  user_role text not null
+  is_bot boolean not null
 );
 
 create table if not exists products (
```

Comment:
rename please tables 'consumer' and 'users_to_role' to 'consumers' and 'users_to_roles'